how-to/auto-complete.md

```diff
@@ -30,43 +28,15 @@ Auto complete candidate terms are stored into `term`. Readings of `term` are sto
 Here is the sample index definition:
 
 ```sql
-CREATE INDEX pgroonga_terms_prefix_search ON terms USING pgroonga
-  (term pgroonga_text_term_search_ops_v2,
-   readings pgroonga_text_array_term_search_ops_v2);
```

Comment:
`readings pgroonga_text_array_term_search_ops_v2` is needed for prefix RK search.